g3/main.py

```diff
@@ -13,6 +16,21 @@ class MessageTone(str, Enum):
     FUNNY = "funny"
 
 
+@app.command()
+def configure() -> None:
+    """Configure G3"""
+    github_token = typer.prompt("🐙 GitHub token", default=config.get("credentials", "github_token"))
+    openai_key = typer.prompt("✨ OpenAI key", default=config.get("credentials", "openai_key"))
+    tone = typer.prompt("🗣️ Default tone", default=config.get("commit", "tone", default=MessageTone.FRIENDLY.value))
```

Comment:
I would add here all the supported configs too:
```
github_api_key=XXX
openai_api_key=XXX
openai_api_type=openai # [openai, azure]
openai_model=gpt-4
openai_temperature=0
openai_api_version= # Skip to default to the latest. Set 2023-05-15 to stick to the a fixed version 
openai_api_base= # Required in case of azure integration
openai_deployment_id= # Required in case of azure integration
```